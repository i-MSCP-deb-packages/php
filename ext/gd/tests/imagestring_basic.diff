001+ 5eaa043cbfb496954fb398c37cbcd05b
001- d0d2fe757400cb7846b36a8c34b41e4a