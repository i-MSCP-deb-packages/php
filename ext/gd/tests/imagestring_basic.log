
---- EXPECTED OUTPUT
d0d2fe757400cb7846b36a8c34b41e4a
---- ACTUAL OUTPUT
5eaa043cbfb496954fb398c37cbcd05b
---- FAILED
