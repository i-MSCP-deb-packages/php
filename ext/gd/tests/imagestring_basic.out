5eaa043cbfb496954fb398c37cbcd05b